packs/orion/actions/status.yaml

```diff
@@ -0,0 +1,17 @@
+---
+description: "Query Solarwinds Orion for a node's status (i.e. Up/Down)"
+enabled: true
+entry_point: 'status.py'
```

Comment:
Minor style / consistency thing.

We tend to prefix action names which return a single result with `get_` (e.g. `get_status` / `get_node_status`) and actions which return multiple results with `list_`.